components/automate-ui/src/app/entities/service-groups/service-groups.model.ts

```diff
@@ -39,7 +39,7 @@ export interface ServiceGroupFilters {
 }
 
 export interface ServicesFilters {
-  service_group_id?: number;
+  service_group_id?: string;
```

Comment:
Existing interface uses string type: https://github.com/chef/automate/blob/master/components/automate-ui/src/app/entities/service-groups/service-groups.model.ts#L25 which is what the API returns ID as.